evalml/pipelines/utils.py

```diff
@@ -140,6 +148,26 @@ class GeneratedPipeline(base_class):
     return GeneratedPipeline
 
 
+def get_generated_pipeline(problem_type):
```

Comment:
Can we call this `get_generated_pipeline_class` to make it more clear that we're returning a class, not an instance?